src/status_file.c

```diff
@@ -308,9 +308,63 @@ static gboolean save_slot_status_locally(RaucSlot *dest_slot, GError **error)
 	return res;
 }
 
+/**
+ * Returns a new GKeyFile optionally pre-populated form a (shared) key file.
+ *
+ * With keep_prefix, one can define groups to keep in the returned GKeyFile.
+ * All other groups will be removed and can be newly populated.
+ *
+ * @param path Path to key file
+ * @param keep_prefix prefix for groups to keep when loading the file
+ *
+ * @return newly-allocated GKeyFile or NULL on error
+ */
+static GKeyFile* key_file_init_from_shared_status_file(const gchar *path, const gchar *keep_prefix, GError **error)
```

Comment:
The path argument seems unused. Was this intended to be used in unit tests?

Reply:
I guess this is a remnant from refactoring. Cannot remember having something in mind for it. Removed.